plugins/google_drive/komand_google_drive/actions/move_file/action.py

```diff
@@ -0,0 +1,27 @@
+import komand
+from .schema import MoveFileInput, MoveFileOutput, Input, Output, Component
+
+# Custom imports below
+from googleapiclient import errors
+
+
+class MoveFile(komand.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="move_file", description=Component.DESCRIPTION, input=MoveFileInput(), output=MoveFileOutput()
+        )
+
+    def run(self, params={}):
+        file_id = params.get(Input.FILE_ID)
+        folder_id = params.get(Input.FOLDER_ID)
+        try:
+            file_parents = self.connection.service.files().get(fileId=file_id, fields="parents").execute()
+            file_parents = ",".join(file_parents.get("parents", []))
+
+            return {
+                Output.RESULT: self.connection.service.files()
+                .update(fileId=file_id, addParents=folder_id, removeParents=file_parents, fields="id, parents")
+                .execute()
+            }
+        except errors.HttpError as error:
+            raise Exception("An error occurred: %s" % error)
```

Comment:
Use PluginException here

Reply:
@mrinehart-r7 I can't use PluginException because this plugin uses Python 2, but we can upgrade Python and refactor the code if you want.

Reply:
@dsliwinski-r7 - ah, I didn't realize it was still on Python 2. Yes, let's get this updated to Python 3 and the latest plugin runtime (especially since Python 2 is no longer supported!)

Reply:
Hi @mrinehart-r7. I updated Python version and refactored the code.